README.md

```diff
@@ -218,6 +218,38 @@ lambda-cfn save dev
 
 We're happy you want to contribute! Check out [CONTRIBUTING.MD](CONTRIBUTING.MD) for more information.
 
+## Dispatch Integration
```

Comment:
For readability, I'd put the section of dispatch use and the util functions above the contribution instructions.